src/dmd/astenums.d

```diff
@@ -430,3 +430,11 @@ enum PINLINE : ubyte
     never,    /// never inline
     always,   /// always inline
 }
+
+enum FILETYPE : ubyte
```

Comment:
D-style wants this to be capitalized as `FileType`.

Reply:
See the other enums. I'm just following the style of the other ones.

Reply:
There's also `InitKind`, `VarArg` and `AliasThisRec`.

Reply:
I agree with @dkorpel

Reply:
Didn't know about those. Done.

Reply:
Also following the style of a given file over the ostensible style of the project (i.e. D style) is not good practice. I am guilty of it myself but it's better to update the old than add more bad practice.

Reply:
Right. I agree with that. I can propose a change on the old ones.